ophyd/areadetector/detectors.py

```diff
@@ -125,6 +126,11 @@ class FirewireWinDetector(DetectorBase):
     cam = C(cam.FirewireWinDetectorCam, 'cam1:')
 
 
+class GreatEyesDetector(DetectorBase):
+    _html_docs = ['greateyesDoc.html']  # the documentation is not public
```

Comment:
```suggestion
    _html_docs = []  # the documentation is not public
```

The documentation is used internally to get docstrings for AreaDetector components, so clearing this list would be appropriate.

Everything else looks fine. Thanks!

Reply:
Thanks @klauer! I tried to apply the suggestion, but don't seem to have permissions to push to @EliotGann's PR branch.

@EliotGann, can you please accept Ken's suggestions on this page? That will go as a separate commit into your branch.

Reply:
I tried to accept the suggestion ... not sure if I did it correctly

Reply:
Yeah, it was correctly done, we can see 583bb9b1181905fe04c088d8db37272670fb402d commit now. Thanks!